atom-table.html

```diff
@@ -265,6 +271,18 @@
         this.$.grid.clearCache();
       },
 
+      _transformTable: function () {
+        var tBody = this.$.grid.querySelector('tbody');
+        if (tBody) {
+          var trItems = tBody.querySelectorAll('tr');
+          if (trItems && trItems.length > 0) {
+            trItems.forEach(function (item, index) {
+              item.style.transform = "translate(0px, " + (index) * 49 + "px)";
```

Comment:
We should consider using some variable instead of hardcoded value "49" later on. Merging the branch for now.